Exercises/if-else-switch/if-else-switch.js

```diff
@@ -4,11 +4,47 @@ const codeToMissingComplexityElement = {
   2: 'special character',
   3: 'cannot be empty'
 };
+let specialCharacterCheck = false;
+const specialCharacters = ['!', '_', '-', '%', '(', ')', '#', '*', '{', '}'];
+let arrayResult = [];
 
 function getMissingComplexityElementCode(password) {
+  for (const char of specialCharacters) {
+    if (password.includes(char)) {
+      specialCharacterCheck = true;
+    }
+  }
```

Comment:
Looks good. You can also try to use `array.some` method here if you want:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some